src/components/PostPageDetail/index.js

```diff
@@ -14,6 +14,7 @@ import {
 import Toast from 'react-native-simple-toast';
 
 import {Footer} from '..';
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement on this line seems to be removed. If the `Context` and related functions are not used in this file, then it's fine. However, if they are still being used, this could cause a reference error.
```diff
+ import {Context} from '../../context';
+ import {saveComment} from '../../context/actions/comment';
+ import {setFeedByIndex} from '../../context/actions/feeds';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->